docs/source-pytorch/extensions/logging.rst

```diff
@@ -131,7 +131,7 @@ the correct logging mode for you. Of course you can override the default behavio
 .. code-block:: python
 
     def training_step(self, batch, batch_idx):
-        self.log("my_loss", loss, on_step=True, on_epoch=True, prog_bar=True, logger=True)
+        self.log("my_loss", loss, on_step=None, on_epoch=None, prog_bar=False, logger=True)
```

Comment:
The section tries to explain that users are able to change the default behaviour. Setting them to `None` matches the signature of the method, however, they have different default values depending on where the method is called from as described in [this table](https://pytorch-lightning.readthedocs.io/en/latest/extensions/logging.html#id3), so I would keep it as is. What do you think?